schema-derivatives-options.graphql

```diff
@@ -1269,36 +1323,27 @@ type ActiveAccount @entity {
 # Note: A position snapshot is not necessary because the position is opened and closed with no other interaction
 # All of the data we would get from a snapshot is already available in the position.
 type Position @entity {
-  " { Account address }-{ Pool address }-{ Position Side }-{ Counter } "
+  " { Account address }-{ Option id }-{ Counter } "
   id: Bytes!
 
   ##### Metadata #####
 
   " The liquidity pool this options contract belongs to "
   pool: LiquidityPool!
 
-  " The type of option this position is. e.g. put/call "
-  type: OptionType!
+  " Reference to the option taken "
+  option: Option!
 
   " Account that owns this position"
   account: Account!
 
-  " The asset the contract was created for (generally the same as the underlying asset in the pool) "
-  asset: Token!
-
-  " The block number this contract expires (maturity) "
-  expirationBlockNumber: BigInt!
```

Comment:
I am guessing this was the same as `takenBlockNumber` that is why we are removing?

Reply:
No, `takenBlockNumber` is when the contract is taken, i.e when the position was created. Expiration block number was moved under Option.